packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -212,21 +212,24 @@ class MiniCart extends Component {
     };
 
     get miniCartInner() {
-        const { checkout, productList, props } = this;
+        const { checkout, productConfirm, productList, props, state } = this;
         const { classes, isCartEmpty, isMiniCartMaskOpen } = props;
+        const { isEditPanelOpen } = state;
 
         if (isCartEmpty) {
             return <EmptyMiniCart />;
         }
 
+        const footer = isEditPanelOpen ? productConfirm : checkout;
```

Comment:
I'm sure that the intention here was that `isEditPanelOpen` could be `true` or `false` but as mentioned above, this value is currently always `false`.